build.gradle.kts

```diff
@@ -67,7 +67,9 @@ extra.apply {
       "--main-jar", "${project.name}-${project.version}-all.jar",
       "--app-version", project.version as String,
       "--copyright", "Copyright © 2001–" + year + " Carl Burch, BFH, HEIG-VD, HEPIA, Holy Cross, et al.",
-      "--dest", "${buildDir}/dist"
+      "--dest", "${buildDir}/dist",
+      "--description", "Digital logic designer and simulator",
```

Comment:
"Digital logic designer and simulator" hears like it designs itself, I would change it in:
"Digital logic design tool and simulator", or something alike

Reply:
I agree "design tool" sounds better.

Reply:
Fixed but:

![ss](https://user-images.githubusercontent.com/8041294/130357404-4dfb81a1-6d34-42d2-86ae-5a406e978a2e.png)

Reply:
I fixed that one :-)